File: Advanced Web Technology/Lab/Lab-5 20-08 Assignment-5/Task5.php
<?php

$simpleInterest = function($principal, $rate, $time) {
    return ($principal * $rate * $time) / 100;
};


$P = 10000;  
$R = 5;      
$T = 2;      

echo "Simple Interest = " . $simpleInterest($P, $R, $T);
?>

File: 2025-26/SEM4/Object Oriented Programming using C++/Lab/Solutions/Lab-9 16-03-26 Assignment 9/task5.cpp
// Write a CPP program to find simple interest using dynamic
//     initialization of object concept

#include <iostream>
using namespace std;
class SimpleInterest
{
    double principal, rate, time;

public:
    SimpleInterest(double p, double r, double t)
    {
        principal = p;
        rate = r;
        time = t;
    }
    double calculateInterest()
    {
        return (principal * rate * time) / 100;
    }
};

int main()
{
    double principal, rate, time;
    cout << "Enter principal amount, rate of interest and time (in years): ";
    cin >> principal >> rate >> time;
    SimpleInterest si(principal, rate, time);
    double interest = si.calculateInterest();
    cout << "Simple Interest: " << interest << endl;
    return 0;
}